test/db/extras/asm_ghidra

```diff
@@ -0,0 +1,725 @@
+            0x000007f3      4889c7         MOV RDI,RAX
+            0x000007f6      e875feffff     CALL sym.imp.malloc
+            0x000007fb      488945f8       MOV qword ptr [RBP + -0x8],RAX
+            0x000007ff      48837df800     CMP qword ptr [RBP + -0x8],0x0
+            0x00000804      743e           JZ 0x844
+            0x00000806      488b55e8       MOV RDX,qword ptr [RBP + -0x18]
+            0x0000080a      488b45f8       MOV RAX,qword ptr [RBP + -0x8]
+            0x0000080e      4889d6         MOV RSI,RDX
+            0x00000811      4889c7         MOV RDI,RAX
+            0x00000814      e827feffff     CALL sym.imp.strcpy
+            0x00000819      488b55f0       MOV RDX,qword ptr [RBP + -0x10]
+            0x0000081d      488b45f8       MOV RAX,qword ptr [RBP + -0x8]
+            0x00000821      4889d6         MOV RSI,RDX
+            0x00000824      4889c7         MOV RDI,RAX
+            0x00000827      e854feffff     CALL sym.imp.strcat
+            0x0000082c      488b45f8       MOV RAX,qword ptr [RBP + -0x8]
+            0x00000830      4889c7         MOV RDI,RAX
+            0x00000833      e818feffff     CALL sym.imp.puts
+            0x00000838      488b45f8       MOV RAX,qword ptr [RBP + -0x8]
+            0x0000083c      4889c7         MOV RDI,RAX
+            0x0000083f      e8ecfdffff     CALL sym.imp.free
+            0x00000844      90             NOP
+            0x00000845      c9             LEAVE
+            0x00000846      c3             RET
+EOF
+CMDS=<<EOF
+e asm.arch=r2ghidra
+e asm.cpu=x86:LE:64:default:gcc
+s main
+pd 44
+EOF
+RUN
+
+NAME=x86_16
+FILE=r2-testbins/mz/unzip.exe
+EXPECT=<<EOF
+            ;-- entry0:
+            0x000156a0      8be8           MOV BP,AX
+            0x000156a2      8cc0           MOV AX,ES
+            0x000156a4      051000         ADD AX,0x10
+            0x000156a7      0e             PUSH CS
+            0x000156a8      1f             POP DS
+            0x000156a9      a30400         MOV [0x4],AX
+            0x000156ac      03060c00       ADD AX,word ptr [0xc]
+            0x000156b0      8ec0           MOV ES,AX
+            0x000156b2      8b0e0600       MOV CX,word ptr [0x6]
+            0x000156b6      8bf9           MOV DI,CX
+            0x000156b8      4f             DEC DI
+            0x000156b9      8bf7           MOV SI,DI
+            0x000156bb      fd             STD
+            0x000156bc      f3a4           MOVSB.REP ES:DI,SI
+            0x000156be      50             PUSH AX
+            0x000156bf      b83400         MOV AX,0x34
+            0x000156c2      50             PUSH AX
+            0x000156c3      cb             RET
+EOF
+CMDS=<<EOF
+e asm.arch=r2ghidra
+e asm.cpu=x86:LE:16:Real Mode:
+pd 18
+EOF
+RUN
+
+NAME=ARM_LE_32
+FILE=r2-testbins/elf/analysis/dwarf_load
+EXPECT=<<EOF
+            ;-- main:
+            0x0000dac8      0dc0a0e1       cpy r12,sp
+            0x0000dacc      10d82de9       stmdb sp!,{r4 r11 r12 lr pc}
```

Comment:
On my local setup, this test fails with the output on this line being:
```
0x0000dacc      10d82de9       stmdb sp!,{ r4 r11 r12 lr pc}
```
Notice the space before `r4`. Any idea what might cause this? I think I should try to run with valgrind to check for uninitialized memory...

Reply:
No idea. But that string is generate by `str = r_str_newf("%s %s", mnem.c_str(), body.c_str());`
So, `sp!,{ r4 r11 r12 lr pc}` should be ths body part. I think this error comes from inside the ghidra.

Reply:
You can try core_ghidra.so to disam it and check outputs.